spec/models/token_spec.rb

```diff
@@ -91,10 +94,14 @@
   end
 
   context "configuration" do
-    before{ allow(token).to receive(:jwt_config).and_return({ 'secret_key' => '123456', 'hmac_sha_algo' => 'SECURE', 'issuer' => 'ME' }) }
-    it{ expect( token.send(:secret_key) ).to eql( '123456' ) }
-    it{ expect( token.send(:hmac_sha_algo) ).to eql( 'SECURE' ) }
-    it{ expect( token.send(:issuer) ).to eql( 'ME' ) }
+    before{ allow(token).to receive(:jwt_config).and_return({
+      "secret_key" => "123456",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.